.github/workflows/release.yml

```diff
@@ -188,3 +188,39 @@ jobs:
             ${{ steps.rename_binary.outputs.name }}.tar.gz
             ${{ steps.rename_binary.outputs.name }}.tar.gz.sha256
             assets/Output/*
+
+  publish-winget:
+    needs: release
+    runs-on: windows-latest
+    steps:
+      - if: ${{ github.event_name != 'workflow_run' }}
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: RussellBanks.Komac
+          installers-regex: 'KomacSetup.*\.exe$'
+          token: ${{ secrets.WINGET_TOKEN }}
+      - if: ${{ github.event_name == 'workflow_run' }}
+        name: Get nightly version
+        run: |
+          $cargo = Invoke-WebRequest -Uri https://raw.githubusercontent.com/russellbanks/Komac/main/Cargo.toml
```

Comment:
This line here feels a bit hacky and might be susceptible to breaking if the toml file changes much